upbound_uxp/manifest.json

```diff
@@ -0,0 +1,76 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "f1427ca4-4b88-484a-a7c5-9cc66542b0d9",
+  "app_id": "upbound-uxp",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Collect and graph Upbound UXP metrics",
+    "title": "Upbound UXP",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Upbound UXP Overview Dashboard",
+        "image_url": "images/upbound_uxp_overview.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Upbound UXP Min Set Dashboard",
+        "image_url": "images/upbound_uxp_min_set.png"
+      }
+
+    ],
+    "classifier_tags": [
+      "Category::AWS",
+      "Category::Azure",
```

Comment:
```suggestion
```
To keep the list of categories focused, I think the cloud category is more relevant than adding the category for each cloud provider. Those are usually reserved for integrations specific to that cloud provider.

Reply:
Removed.